torch/csrc/jit/codegen/cuda/lower2device.cpp

```diff
@@ -266,14 +268,14 @@ void GPULower::replaceSizes() {
   for (TensorView* tv : orig_inp_out) {
     // Replace the domain with one based on Ti.size[j]
     std::vector<IterDomain*> new_domain_iters;
-    TensorDomain* root_td = tv->getRootDomain();
+    const std::vector<IterDomain*>& root_td = tv->getRootDomain();
 
-    for (decltype(root_td->nDims()) i{0}; i < root_td->nDims(); i++) {
+    for (decltype(root_td.size()) i{0}; i < root_td.size(); i++) {
```

Comment:
nit: clean this to: size_t i = 0?

Reply:
Made a note on https://github.com/csarofeen/pytorch/issues/34 to run through and clean decltypes up.